### PR TITLE
Update intune-notices.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to state  " Apple has released iOS 15. Microsoft Intune, including the Intune Company Portal and Intune app protection policies (APP, also known as mobile application management), will require  iOS/iPadOS 13 and later." as this content is included in other documents such as  fundamentals/whats-new.md and fundamentals/in-development.md

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/2056

**Comment by PRMerger13** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/includes/intune-notices.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ No action is needed at this time. Review our [In Development](../fundamentals/in
 
 ### Plan for change: Intune is moving to support iOS/iPadOS 13 and later<!--10144130-->
 
-Later this year, we expect Apple to release iOS 15. Microsoft Intune, including the Intune Company Portal and Intune app protection policies (APP, also known as mobile application management), will require  iOS/iPadOS 13 and later shortly after iOS 15's release.
+Apple has released iOS 15. Microsoft Intune, including the Intune Company Portal and Intune app protection policies (APP, also known as mobile application management), will require  iOS/iPadOS 13 and later.
```
Comment (JohanFreelancer9) [inline memdocs/intune/includes/intune-notices.md:127]:
```suggestion
Apple has released iOS 15. Microsoft Intune, including Intune Company Portal and Intune app protection policies (APP, also known as mobile application management), require iOS/iPadOS 13 and later.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@ErikjeMS Please review and merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dougeby Gentle reminder, Please merge this PR. Thanks

**Comment by PRMerger9** [comment]
@dougeby : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**